website/scripts/get-bug-and-pr-report.js

```diff
@@ -376,57 +376,87 @@ module.exports = {
     }, 0);
     // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
+    const kpiResults = [];
+
+    // NOTE: If order of the KPI sheets columns changes, the order values are pushed into this array needs to change, as well.
+    kpiResults.push(
+      averageDaysContributorPullRequestsAreOpenFor,
+      daysSinceContributorPullRequestsWereOpened.length,
+      averageDaysPullRequestsAreOpenFor,
+      daysSincePullRequestsWereOpened.length,
+      averageNumberOfDaysReleasedBugsAreOpenFor,
+      averageNumberOfDaysUnreleasedBugsAreOpenFor,
+      allBugsClosedInPastWeek.length,
+      averageNumberOfDaysBugsAreOpenFor,
+      allBugsCreatedInPastWeek.length,
+      allBugsCreatedInPastWeekEndpointOps.length,
+      allBugsCreatedInPastWeekEndpointOpsCustomerImpacting.length,
+      allBugsCreatedInPastWeekEndpointOpsReleased.length,
+      allBugsCreatedInPastWeekEndpointOpsUnreleased.length,
+      allBugsCreatedInPastWeekMobileDeviceManagement.length,
+      allBugsCreatedInPastWeekMobileDeviceManagementCustomerImpacting.length,
+      allBugsCreatedInPastWeekMobileDeviceManagementReleased.length,
+      allBugsCreatedInPastWeekMobileDeviceManagementUnreleased.length,
+      daysSinceBugsWereOpened.length,
+      allBugsWithReleasedLabel.length,
+      allBugsWithUnreleasedLabel.length);
+
+    console.log(kpiResults.join(','));
+
     // Log the results
     sails.log(`
-    Bugs:
+
+    CSV for copy-pasting into KPI spreadsheet:
     ---------------------------
-    Number of open issues with the "bug" label in fleetdm/fleet: ${daysSinceBugsWereOpened.length}
-    Average open time: ${averageNumberOfDaysBugsAreOpenFor} days.
+    ${kpiResults.join(',')}
 
-    Number of open issues with the "~unreleased bug" label in fleetdm/fleet: ${allBugsWithUnreleasedLabel.length}
-    Average open time: ${averageNumberOfDaysUnreleasedBugsAreOpenFor} days.
+    Pull requests:
```

Comment:
All of the other changes is just adjusting the order they are printed to match the order in the KPI sheets.